### PR TITLE
Tweak grammar and formatting

### PR DESCRIPTION
**Pull request by half0wl**
0. [x] I have read the [Code of Conduct](https://github.com/minelarka14/awesome-singapore/CODE_OF_CONDUCT.md)

**What is your Idea?**
Improve readability by tweaking the grammar of intro paragraph, along with some formatting.

**Which category will your idea fall under?**
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-<img src="https://shields.io/github/stars/minelarka14/awesome-singapore?style=social"/>
-<img src="https://shields.io/github/forks/minelarka14/awesome-singapore?style=social"/>
+A curated list of awesome resources related to Singapore.
 
-<br>
-<br>
+This list only exists because of **YOU**. If you have any suggestions on what to add, please [create a pull request or issue](https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/creating-an-issue-or-pull-request) on this repository. If you're unsure on how to do this, don't worry! You can e-mail the maintainer at [muhammadosaid06@gmail.com](mailto:muhammadosaid06@gmail.com?subject=%5BGITHUB-AWESOME-SG%5D%20New%20Idea%20%3CA%20short%20description%20of%20you%20idea%3E&body=Hello%2C%20I%20have%20a%20new%20Idea%20for%20your%20GitHub%20awesome%20Singapore%20repository.%20%0A%0A--%3E%20Your%20Idea%0A%0A--%3E%20Which%20category%20will%20it%20fall%20under%0A%0A%0A).
 
-> A curated list of awesome resources related to Singapore.
+### Maintainer's note:
```
Comment (half0wl) [inline README.md:9]: I took the liberty of modifying this section a bit: I think it's important to have a distinction between a maintainer's note/request and the project. Open to suggestions here!

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-
-> I spent quite a while on this. If you could support me by following me on my socials, it would be greatly appreciated! <img src="https://shields.io/github/followers/minelarka14?label=Follow"/> <img src="https://img.shields.io/twitter/follow/minelarka13?style=social"/> and follow me on [Instagram](https://www.instagram.com/minelarka13/). Thank you!
+> I spent quite a while of time on this. If you could support me by following me on my socials, it would be greatly appreciated!
+>
+> <img src="https://shields.io/github/followers/minelarka14?label=Follow" /> <img src="https://img.shields.io/twitter/follow/minelarka13?style=social" /> <a href="https://www.instagram.com/minelarka13/"><img src="https://img.shields.io/badge/Instagram-@minelarka13-E4405F?style=for-the-badge&logo=instagram&logoColor=white" /></a>
```
Comment (half0wl) [inline README.md:13]: (personally not a fan of the IG badge; I couldn't find a way to tweak its res, so it stands out quite a bit)

**Comment by half0wl** [comment]
@minelarka14 tagging you for a review request

**Comment by minelarka14** [comment]
Excellent PR, thank you so much!

**Pull request merged**